src/components/SuperDrawer/SuperDrawerContent.tsx

```diff
@@ -29,6 +30,9 @@ export const SuperDrawerContent = ({ children, actions }: SuperDrawerContentProp
 
   // Determine if the current element is a new content element or an detail element
   const { kind } = contentStack.current[contentStack.current.length - 1] ?? {};
+  const firstContent = contentStack.current[0]?.opts as OpenInDrawerOpts;
+
+  const width = (firstContent && firstContent.width) || SuperDrawerWidth.Normal;
```

Comment:
**nit (non-blocking):** I don't have a problem with this, but sometimes seeing a mix of `&&` and `||` operators makes reading more difficult in my opinion. I probably would've done something more like: 
```
const { width = SuperDrawerWidth.Normal } = firstContent ?? {};
```
Again, non-blocking, but just wanted to provide an alternative way to define the `width`.

Reply:
Forgot to change, thanks